bigquery/indexer.py

```diff
@@ -210,6 +207,23 @@ def index_table(es, index_name, table_name, participant_id_col, sample_id_col,
     logger.info('pandas -> ElasticSearch index took %s' % elapsed_time_str)
 
 
+def index_fields(es, index_name, table):
+    field_docs = _field_docs_by_id(
+        _get_table_name(table.full_table_id), table.schema)
+    indexer_util.bulk_index_docs(es, index_name, field_docs)
+
+
+def read_table(client, table_name):
+    project_id, dataset_id, table_name = table_name.split('.')
+    return client.get_table(
+        client.dataset(dataset_id, project=project_id).table(table_name))
+
+
+def _get_table_name(legacy_table_name):
```

Comment:
By convention, helper methods go before where they're used. Can you move this to before _create_nested_mappings?